mix.exs

```diff
@@ -25,7 +25,7 @@ defmodule UeberauthGoogle.Mixfile do
 
   defp deps do
     [{:ueberauth, "~> 0.4"},
-     {:oauth2, "0.6.0"},
+     {:oauth2, "~> 0.8.0"},
```

Comment:
Let's bump this to `0.8.1`

That will allow folks to take advantage of the new debug option.